docs/src/man/seq.md

```diff
@@ -297,7 +297,99 @@ julia> char"αβγδϵ"
 
 ```
 
-Sequence can also be constructed from strings or arrays of nucleotide or amino
+However it should be noted that by default these sequence literals
+allocate the BioSequence object before runtime.
+This means there may be occasions where your program does not behave as you
+first expect, even though it is the intended behaviour.
+For example consider the following code:
+
+```jlcon
+julia> using Bio.Seq
+
+julia> function foo()
+           s = dna"CTT"
+           push!(s, DNA_A)
+       end
+foo (generic function with 1 method)
+```
+
+You might expect that every time you call `foo`, that a DNA sequence `CTTA` would
+be returned. You might expect that this is because every time `foo` is called,
+a new DNA sequence variable `CTT` is created, and and `A` nucleotide is pushed
+to it, and the result, `CTTA` is returned.
+In other words you might expect the following output:
+
+```jlcon
+julia> foo()
+1nt DNA Sequence:
+CTTA
+
+julia> foo()
+2nt DNA Sequence:
+CTTA
+
+julia> foo()
+3nt DNA Sequence:
+CTTA
+```
+
+However, this is not what happens, instead the following happens:
+
+```jlcon
+julia> foo()
+1nt DNA Sequence:
+CTTA
+
+julia> foo()
+2nt DNA Sequence:
+CTTAA
+
+julia> foo()
+3nt DNA Sequence:
+CTTAAA
+```
+
+The reason is because the sequence literal is allocated before runtime, `s` in
+`foo` is always a reference to that one sequence that was allocated before the
+runtime of your program.
+So one sequence is created, and then pushed to every time `foo` is called. Thus
+that one allocated sequence grows with every call of `foo`.
+
+If you wanted `foo` to create a new sequence each time it is called,
+the you can add a flag to the end of the sequence literal: a flag of 's'
+means the sequence will be allocated before runtime as is the default behaviour.
+However providing 'd' as the flag will make sure that the sequence is
+allocated at runtime. So to change `foo` so as it creates a new sequence
+each time it is called, simply add the 'd' flag to the sequence literal:
```

Comment:
I think we should, as a mnemonic, mention 's' and 'd' represents *static* and *dynamic*, respectively.